batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/validator/EmployeeValidator.java

```diff
@@ -0,0 +1,59 @@
+package com.lftechnology.batch7crud.validator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import javax.validation.ValidationException;
+
+import com.lftechnology.batch7crud.constants.EmployeeAttributeConstants;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.lftechnology.batch7crud.constants.EmployeeAttributeConstants'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)